### PR TITLE
footer

### PR DESCRIPTION
**Pull request by mariangz**

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+}
+
+body {
+  font-family: $font-main;
+}
+
+.container {
+  display: grid;
+  max-width: 1200px;
+  min-width: 100%;
```
Comment (AlbaHerrerias) [inline src/App.scss:27]: If I'm not mistaken, you're using `min-width: 100%` to center the content. You can center doing `margin: 0 auto`, which semantically makes more sense. No need to change, just some thought I wanted to share 😄

File: src/App.scss [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    justify-content: space-between;
+    ul {
+      display: flex;
+    }
+    span {
+      display: none;
+    }
+  }
+
+  div:nth-child(6) {
```
Comment (AlbaHerrerias) [inline src/App.scss:76]: Just some thought, no need to change: the problem with hardcoding values like this is that, if I ask you to insert another column, this code will break and would need a refactor. In these cases I would add a class, it is also easier for the developer to spot which one it's been affected by these styles.

File: src/Footer.js [hunk 3]
```diff
@@ -1,85 +1,185 @@
...
+              <FontAwesomeIcon icon={brands('instagram')} />
+              <span>Instagram</span>
             </a>
           </li>
           <li></li>
         </ul>
       </div>
       <div>
-        <h6>About</h6>
+        <h6 className='title'>Use Cases</h6>
```
Comment (AlbaHerrerias) [inline src/Footer.js:38]: Another thought, no need to change: I think I would have created an object like 
```
const columns = [
  {title: "Use Cases", links: [{ label: UI design, ...}, ...]},
  ...
];
```

And iterate over it. I think it is more resilient to changes.

**Review by AlbaHerrerias — APPROVED** [review]
Very good! ✨  It looks really smooth and the goal of the exercise it's completed 😄  Some tiny notes that I don't think are worth addressing as there are more important topics you should focus on.

File: src/App.scss (hunk 1 above)
Comment (mariangz) [reply src/App.scss:27]: super! I'll change that!

File: src/App.scss (hunk 2 above)
Comment (mariangz) [reply src/App.scss:76]: Right! I think you told me something like that when we talked about margin, thanks

File: src/Footer.js (hunk 3 above)
Comment (mariangz) [reply src/Footer.js:38]: Oh, with React! Great!

**Pull request merged**